### PR TITLE
Add edit xml functionality on license request page 

### PR DESCRIPTION
**Pull request by tjasmith**
For issue #76

### CODE REVIEW

**Comment by tjasmith** [comment]
@techytushar @rtgdk @goneall Please do not review yet, as I am working on a way to make this solution better.

**Comment by tjasmith** [comment]
@techytushar You can now review. I have updated the PR.

Thanks.

**Comment by tjasmith** [comment]
@rtgdk @techytushar 
In case of an error, I redirect to the 404 page.
I have updated my branch with the latest version of the repo.
I have added tests on the edit xml functionality, and here is a screenshot of the tests as they pass:
tests2

File: src/app/urls.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
     url(r'^checkusername/$', views.checkusername, name='check-username'),
     url(r'^xml_upload/$',views.xml_upload, name='xml-upload'),
     url(r'^edit/(?P<page_id>[0-9a-z]+)/$', views.xml_edit, name='editor'),
+    url(r'^edit_license_xml/(?P<license_id>[0-9]+)/$', views.edit_license_xml, name='license_xml_editor'),
```
Comment (techytushar) [inline src/app/urls.py]: I would suggest you to put `*` instead of `+` at that Regex, then only the redirect would work

**Comment by techytushar** [comment]
Just a small change otherwise LGTM, great work :+1:

File: src/app/urls.py (hunk 1 above)
Comment (tjasmith) [reply src/app/urls.py]: @techytushar 
I have added the changes you kindly requested.

**Comment by goneall** [comment]
Tested this out locally - merging into the master and deploying.

@tjasmith Thanks for the contribution!

**Pull request merged**